docs/tutorials/google/spin_echoes.ipynb

```diff
@@ -0,0 +1,1362 @@
+{
```

Comment:
<p>For the sake of completeness, can we print out the gateset of the processor by querying the engine? There's also another implicit assumption in the cell below that <code>PhX</code>  gates are part of the target gateset. It would be nice to make it explicit what the target gateset actually is. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4373/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Done!</p>